site/docsource/Get_started.adoc

```diff
@@ -2,22 +2,37 @@
 
 ### First example
 
-* Create a directory called `hello` and create `package.json`
-  as below:
+* Create a directory called `hello`
+* Init the project as a NPM package and fill-in the details
+----
+npm init
+----
+* Add bucklescript as a dev dependency
+-----
+npm install --save-dev bs-platform
+-----
+* In the package.json, remove the `test` script and add a `build` script. You're file should look like:
 +
 [source,js]
 .package.json
 ----
 {
-    "dependencies": {
-        "bs-platform": "0.9.3" // <1>
+  {
```

Comment:
I get the intent behind displaying a minimal script, and it seems reasonable. However, it is very unlikely that anyone would edit it by hand, unless for modifying the scripts. Well for sure, I wouldn't add a dependency myself :)

I can change it back, if you really prefer :) What about using npm for adding the dependency, then?